### DISCUSSION TITLE
mhm

### DISCUSSION THREAD

**Discussion by Outmoddedsoup436** [remarkablegames/remarkablegames.github.io#55] [General]
Back in time to the middle of a waging war between rome and egypt in 323 B.C
There was a man named Tome, he was a normal person for the most part, didn’t have any special skills or anything. But even though that, he still had to enter the war. He didn’t want to go into the war,
 “No, I refuse to be drafted into this war”, he said, 
“But you have to, Tome. This will help your mother, I know of your money situation right now. This could be what you need,” another voice exclaimed.
“This is my uncle Jared, a renowned commander in the war of 268 B.C, pretty much the only family I have left, he couldn’t be drafted into this war for having scars both physical and mental. But he wants some way to get into the war, to get in some way, me. He’s just using me isn’t he? No, he’s right I do need this, it will help me pay to help my mom for her sickness, but I don’t want to go into this, just in case I die and she's alone.” Tome spoke to himself. 
“Look, I know you’ll choose correctly, just do it quickly. Remember it's a family tradition” Jared said. 
“I know” said Tome, 
“Hey remember, choose before it gets chosen for you”. Added Jared while walking away.
 “That's right, this is the family tradition. My family is known for being very exceptional in combat, and we are known for joining into battles and war, but i’m the only one who doesn’t have any special skills, yet for some reason my uncle thinks that I have some hidden talents, something to keep this tradition alive, some way to keep this family alive. But I don't know, my mom is sick, and I have no one to watch her if I leave. I just can’t go or she might die alone. But, if I do leave and she survives for me to come back, if I, survive to come back, then the pros would help her 10 fold ...... Dang, I don't know what to do.” Tome solemnly spoke to himself. Tome starts walking away in a contemplating and thinking state, “I need to get back home” he says. Tome walks and walks until he finds his street, seeing his house he starts walking faster and faster till he runs in to say, “Hey mom, Im home!” he starts looking for his mom, “mom?” he says “in here!” another voice says. Tome thinks to himself; “that's the kitchen” he starts walking towards the kitchen, “Mom??” 
“Hey Tome” 
“What are you doing in the kitchen??” 
	“Well, I just thou-”
	“No, mom, I said you should be laying in bed. I’ll be the chef while your sick”
	“But-” 
	“You know the rules mom. Now you should go to bed”
	“Ok. Your right” says his mom 
“Man, how many times am I going to have to tell her”, he says to himself. Tome begins finishing the food his mom was already cooking, he lays out a plate, a fork and a spoon on the table, he makes a plate for his mom and takes it up to her, “Hey, the food is done.” he says knocking on the door. But no one answers, so he opens the door to find his mom already asleep, he puts the food down near her and says, “Goodnight, mom.” Tome goes back downstairs, eats his food and goes to bed. “Tomorrow will be a better day” Tome states, falling fast asleep, Tome wakes up in the middle of the night to a horrifying scream, “Mom!” he says, “Mom?!!? Are you ok!!?!” He rushes quickly to her room, not see her there he starts heading down the stairs seeing his mom laying on the floor coughing blood with sores upon other things on her, “Oh no, it's getting much worse” Tome states, “We have to get to the hospital now” After saying that he picks her up and starts charging as fast as possible to the hospital, “Oh no” “Oh no” he starts things to himself “She can’t die, she cant!” He reaches the hospital “Hello?!” he calls out, a lady and some others come out of their offices, seeing his mom they rush to get her to a place where they can check on her, “Is she going to be ok?” Tome says helping them rush to a room, “We don’t know.” says one of the doctors. They get her to a room and start making sure she's ok, checking on her, 
“Well?”  
“We don’t know yet. Come back in the morning once we’ve had enough time to give you an answer. Trust me.” 
	“Ok. I do, thank you.” Tome starts walking away leaving it in their hands for the night. “This can’t be any good, I don’t think I'm even going to get any sleep tonight.” “dang” -Tome speaks to himself, “ Atleast, it was doctor Grehk, he’s someone I can trust with this, he’s capable and has helped me and my mom before.”, “I’m sure he’ll fix this and get it all sorted out.”, Tome walks home and tries to sleep that night, but it seems Tome was correct with his earlier statement “I don’t think I'm even going to get any sleep tonight.”, The next morning he heads to the hospital searching for his mom, “Time to go see my mom. Hopefully she's ok” Tome finally reaches the hospital after walking so far “Ok, I think it's time to go in '' he heads inside calling for doctor Grehk, “Doctor Grehk? I came to see my mom, hello?” 
“Hello Tome sorry for the, wait”
	“Its ok” 
“How’s my mom?”
	“It’s not looking so good. Come see for yourself”
Tome speaks in his head. “Oh no, this can’t be happening”
Dr. Grehk takes Tome to the room his mother was put in “This way, Tome.”
	“Ok, I think I can do this”
	“I sure hope so, it’s not good.”
Dr. Grehk opened the door for Tome, “Oh my god…” 
	“Yeah, it’s pretty bad.” 
	“Is she gonna be ok??”
	“Well, no. But”
	“But?? But what?! How can I save her??!”
	“There is a treatment that could help her, but it’s only in development. 
	“I’LL DO IT”
“But we don’t know if it will fully help her.”
“Well if there's a way, I’LL DO IT. WHATEVER THE COST.”
“Well, then you can. But since it’s in development it does cost a lot…”
“How much?” 
“Sadly more than you have.”
“Oh no.. no no NO” 
“I’m sorry but I can’t help the cost. You’re going to have to find a way to get the money quickly. She could have months, weeks, days. And even hours.”
“But there's no job that much and that quickly.”
“Well, theirs 1” Dr. Grehk shows him the flier to join the soldiers for the war.
“You only need to be in service for about 1 to 6 months to earn enough.”
“That’s too long, I know I can help her myself. Let me take her home.”
“I’m sorry but I can’t do that because she is a patient in the emergency services, its my duty as a Doctor to keep her here. I can’t let you do that”
“Dang it.”
Tome speaks to himself, “I should listen to Dr. Grehk. He usually knows what's best, but the soldier thing, I don’t know if it’s my best option at the moment.” “No No, it isn’t, I need him!” Tome speaks aloud “No There has to be another option, I can’t leave for war knowing that i’ll probably die!”
“ I see, well if you're that opposed to it. Than maybe…”
“ What? Than maybe what?”
“ Well, it still requires you to leave for a while”
“What is it?”
“You can go out and find the specific herb and or herbs needed for it, instead of going out to war for months just to maybe get paid enough”
“I’ll do it” 
“But this may take even longer, it’s a pretty rare herb”
“It doesn’t matter if there's a chance to get here faster than i’ll take it.”
“Ok, well here is the herb you're looking for, it’s called luminescent eyeity (pronounced I it e) it has a bluish dark green glow usually found in dark areas and only grows in the dark.”
“Oh no wonder it so rare”
“Yes this is what I was talking about. But if you really want to do it you can do it.”
“Yes, I’ll do it, is their a good area that you know of?”
“No, but like I said before, it can be found in darker areas, so cavern and other areas. Also you might run into others trying to get a hold of it, it can be used for many things besides medicine.” 
“Well, if that's all you have for me I will have to get going. Thanks\ again for this, it will help me and my mom so much.”
“You’re welcome, and be careful.”
Tome starts running back to his house to get ready, “I gotta hurry up, I don’t know how long my mom has left, so I just have to pack my essentials and go.” he gets home and packs all of his stuff, “Ok I got everything I need. Gotta go fast” Tome heads to the gate of the town,” well here goes nothing” Tome speaks aloud, 
“I knew it!” Tome turns to see Jared saying that; “I knew you would choose the right choice.” 
“No no, ugh nevermind i’ll tell him when I get back”
Tome leaves the town heading on his journey to find this herb “Well thankfully I know the area, I think there's some good places where I could find it this way, Tome travels out of the line of sight of the town getting further and further, “I don’t know how long this will take but maybe, just maybe I can get it and save my mom.”

**Comment by remarkablemark** [comment]
Epic
